samples/hosting/docker/sample.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Hosting endpoints in Docker containers
+summary: Hosting multiple endpoints in several Docker containers managed by Docker Compose
+reviewed: 2017-09-22
+component: Core
+tags:
+- Hosting
+- Docker
+related:
+- nservicebus/hosting
+---
+
+## Prerequisites
+
+This sample requires that the following tools are installed:
+
+  - .NET Core 2.0 SDK (link TBD)
+  - Docker (link TBD)
+
+## Building and publishing binaries
+
+First step is to build the binaries using the .NET Core command line tools:
+
+```
+dotnet build
```

Comment:
why `dotnet build`. we dont use this in any other samples?

Reply:
The whole flow at the comment is command line based. The idea is to keep the reader at the command line and not flipping between VS and command line.

Reply:
Should the `build` and `publish` commands be passing `-c Release` since they default to Debug builds instead?

Reply:
all samples default to debug and only have debug config

Reply:
That was the goal. Release configurations will be removed from the sln file.